src/main/kotlin/no/digdir/informasjonsforvaltning/fdk_dataset_harvester/controller/CatalogsController.kt

```diff
@@ -0,0 +1,38 @@
+package no.digdir.informasjonsforvaltning.fdk_dataset_harvester.controller
+
+import no.digdir.informasjonsforvaltning.fdk_dataset_harvester.generated.api.DcatApNoCatalogsApi
+import no.digdir.informasjonsforvaltning.fdk_dataset_harvester.rdf.JenaType
+import no.digdir.informasjonsforvaltning.fdk_dataset_harvester.rdf.jenaTypeFromAcceptHeader
+import no.digdir.informasjonsforvaltning.fdk_dataset_harvester.service.CatalogService
+import org.slf4j.LoggerFactory
+import org.springframework.http.HttpStatus
+import org.springframework.http.ResponseEntity
+import org.springframework.stereotype.Controller
+import javax.servlet.http.HttpServletRequest
+
+private val LOGGER = LoggerFactory.getLogger(CatalogsController::class.java)
```

Comment:
Kan erstattes med Sl4j annotasjon overalt

Reply:
Kan, men... 
Tar ikke inn lombok for å bytte ut 1 linje kode med en annotasjon